packages/core/tests/integrations/fr.test.ts

```diff
@@ -13,92 +13,94 @@ test("string parser error messages", () => {
 
   expect(getErrorMessage(schema.safeParse(undefined))).toEqual("Obligatoire");
   expect(getErrorMessage(schema.safeParse(1))).toEqual(
-    "Type invalide: chaîne de caractères doit être fourni(e), mais nombre a été reçu(e)"
+    "« chaîne de caractères » attendu mais « nombre » a été reçu"
```

Comment:
attendue ?

Reply:
Here I think we should use the genre of type (type chaîne, type nombre, etc) that's why I use quotes.

Reply:
For consistency, we could use texte instead of chaîne de caractères

Reply:
I'm not a French speaker so I don't know what this is a discussion about, but am I correct in assuming that this has already been resolved? Or is it still open for discussion?

Reply:
Small update to include review:
- use the word "chaîne" everywhere for consistency
- add "Le type " to avoid ambiguity about grammatical agreement